Cognigy/WebchatWidget
# Fix security vulnerabilities

npm audit has shown a critical vulnerability from a dependency of dependencies for a package called `minimist`. Since this package is a dependency of lots of packages I have rebuilt the package-lock.json to update them all. This resolved the critical vulnerability as well as some other high ones.

How to test:
run `npm run dev`
in another terminal run `nom cypress:open` and run all the tests and make sure the visuals are also still as expexted.
